javascript-data-model/order-history.js

```diff
@@ -0,0 +1,71 @@
+var orderHistory = [
+  {
+    order: {
+      orderNumber: '114-3941689-8772232',
+      orderDate: 'August 4, 2020',
+      orderTotal: '$34.00',
+      numberOfItems: 1
+    },
+    items: {
+      itemName: ['JavaScript for impatient programmers'],
```

Comment:
Instead of having arrays in here for the values for each object, I would make the `items` property hold an array, and in that array is an object for each item